gen/definitions/policyassignments.yaml

```diff
@@ -0,0 +1,66 @@
+---
+name: Policy Assignments
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/assignment/policyassignments
+no_delete: true
+no_data_source: true
+data_source_name_query: true
+doc_category: Policy Assignments
+attributes:
+  - model_name: type
+    type: String
+    value: "PolicyAssignment"
+  - model_name: name
+    type: String
+    description: User-created name of the resource.
+    mandatory: false
+    example: AccessPolicyName
+    data_path: ["policy"]
+    exclude_example: true
+    exclude_test: true    
+  - model_name: type
+    type: String
+    mandatory: false
+    example: AccessPolicy
+    data_path: ["policy"]
+    exclude_example: true
+    exclude_test: true    
+  - model_name: id
+    tf_name: policy_id
+    type: String
+    id: true
+    mandatory: true
+    example: 0050568A-2561-0ed3-0000-004294972836
+    data_path: ["policy"]
+    test_value: fmc_access_control_policy.example.id
```

Comment:
Description missing. This is the ID of policy to be assigned.

Reply:
Done